lib/barbeque/runner/hako.rb

```diff
@@ -0,0 +1,46 @@
+require 'open3'
+
+module Barbeque
+  module Runner
+    class Hako
+      # @param [Barbeque::DockerImage] docker_image
+      # @param [String] hako_dir
+      # @param [Hash] hako_env
+      # @param [String] yaml_prefix
```

Comment:
I agree that it's a little hard to understand and probably we should use something like yaml_dir (absolute path of yaml_dir + yaml_prefix) instead of yaml_prefix (relative path from hako_dir).